alsongDalsong/alsongDalsong/alsongDalsong/Sources/Views/Game/GameNavigationController.swift

```diff
@@ -38,6 +38,20 @@ final class GameNavigationController: @unchecked Sendable {
                 self?.gameInfo = gameState
             }
             .store(in: &subscriptions)
+        
+        gameStateRepository.receiveKickOut()
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] isKickedOut in
+                guard isKickedOut else { return }
+                self?.leaveRoom()
```

Comment:
무조건 true일때만 무언갈 해야한다면 operator로 위에서 구분해줘서 내려오는 것도 좋을 것 같습니다. (개인적인 의견입니다!)

```suggestion
            .receive(on: DispatchQueue.main)
            .filter { $0 }
            .sink { [weak self] _ in
                self?.leaveRoom()
```

Reply:
반영되었습니다. !